app/scripts/services/dimStoreService.factory.js

```diff
@@ -336,15 +336,16 @@
       console.time('Load stores (Bungie API)');
       _reloadPromise = $q.all([dimDefinitions,
         dimBucketService,
+        dimMissingSources,
         loadNewItems(activePlatform),
         dimItemInfoService(activePlatform),
         dimBungieService.getStores(activePlatform)])
-        .then(function([defs, buckets, newItems, itemInfoService, rawStores]) {
+        .then(function([defs, buckets, missingSources, newItems, itemInfoService, rawStores]) {
           console.timeEnd('Load stores (Bungie API)');
           if (activePlatform !== dimPlatformService.getActive()) {
             throw new Error("Active platform mismatch");
           }
-
+          dimMissingSources = missingSources;
```

Comment:
Reassigning `dimMissingSources` from a promise to its resolved value isn't a great idea. Just pass along the resolved value to where it needs to go.